### DISCUSSION TITLE
Notcurses + SDR Waterfall + Data Decoding: Girls would think we are so cool since we're pretty much assimilated 95% Borg.

### DISCUSSION THREAD

**Discussion by SuperPauly** [dankamongmen/notcurses#2733] [General]
#### He wrote it flawlessly in strings of text... The audacity.

Did anything ever come of it?

It would be cool to make one but like most internet dwellers I used the internet twice and caught ADHD and since have started 3+17**Grahams-number projects and starting 1 more will tip me over the fucking edge.

I know Nicks busy but did anyone make one or start making one?

If so this would be an alpha move. World domination next?

I've had a quick play with them demo and then checked out the examples and, well. I will stick with Python for now..

_This post was done 100% by a living human in the year 2023._

**Comment by dankamongmen** [comment]
i have been working on my [first novel](https://nick-black.com/dankwiki/index.php/Midnight%27s_Simulacra) the past few months, and before that professional stuff for most of the year. i've still got some code on this, basic stuff, but i won't be developing it any further until 2024.
